frontend/js/components/map.js

```diff
@@ -139,15 +138,11 @@ export class LeafletMap extends React.Component {
             return <IssueMarker {...props} />
         })
 
-        let extents = geojsonExtent(JSON.parse(JSON.stringify(geojson)));
```

Comment:
@thammerl Yes, this hack was/is for reasons of immutability, something in the geojsonExtent code changes the actual object. Might be fixed by now